seatunnel-e2e/seatunnel-connector-v2-e2e/connector-paimon-e2e/src/test/java/org/apache/seatunnel/e2e/connector/paimon/PaimonIT.java

```diff
@@ -34,7 +34,7 @@
 import java.nio.file.Path;
 
 @DisabledOnContainer(
-        value = TestContainerId.FLINK_1_13,
+        value = {TestContainerId.FLINK_1_13,TestContainerId.SPARK_3_3},
```

Comment:
Why add this? cc @TyrantLucifer

Reply:
update to disable on flink 1.15

here is the error log. run with flink 1.15. we can see the data all is not null, and from the config we also know the schema is not null. but get NPE.  
Try disabling on Flink 1.15 to see if there are still have error.

https://github.com/liunaijie/seatunnel/actions/runs/5608902041/job/15195542417#step:4:12511

https://github.com/liunaijie/seatunnel/actions/runs/5608902041/job/15195542417#step:4:12626

Reply:
> update to disable on flink 1.15
> 
> here is the error log. run with flink 1.15. we can see the data all is not null, and from the config we also know the schema is not null. but get NPE. Try disabling on Flink 1.15 to see if there are still have error.
> 
> https://github.com/liunaijie/seatunnel/actions/runs/5608902041/job/15195542417#step:4:12511
> 
> https://github.com/liunaijie/seatunnel/actions/runs/5608902041/job/15195542417#step:4:12626

Is the engine not supported?